### ISSUE TITLE
Training hyper-parameters

### ISSUE THREAD

**Issue by adityasoni9998** [InternLM/SWE-Fixer#9]
Hi @yitianlian! Thanks once again for the great work on SWE-Fixer! I was reading your paper and realize that there is no discussion about what were your training hyper-parameters (epochs and learning rate, scheduler, optimizer, etc.) for the experiments? I am mainly interested in the hyper-parameters for the Qwen-2.5 Coder 32B model's fine-tuning. It would be great if you can also share these (and even the hyper-params for other models like Qwen 72B would be super helpful)! Thanks in advance.

**Comment by yitianlian** [comment]
For the 32B models, we fine-tune with 3 epochs, a learning rate of 1e-5/3e-6, and a batch size of 96. For the 72B model, we increase the learning rate from 1e-5 to 2e-5 while keeping all other settings the same.

**Comment by adityasoni9998** [comment]
Thanks a lot for your response!
